pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -302,6 +303,11 @@ func ManageClientCABundle(ctx context.Context, lister corev1listers.ConfigMapLis
 	if err != nil {
 		return nil, false, err
 	}
+	if requiredConfigMap.Annotations == nil {
```

Comment:
I was doing this in my PR on service-ca-operator as well but then I think it is better just to group all of ownership-related annotations operation to a func. Then, insert that func anywhere you need. Plus, it may be handy in the future to add more annotations settings in that func instead of having to coping the code everywhere.

Reply:
Good idea, we should have "set necessary annotations" func and "verify annotations" func